src/main/java/org/jenkinsci/plugins/vsphere/builders/ReconfigureCpu.java

```diff
@@ -91,6 +99,7 @@ public boolean reconfigureCPU (final Run<?, ?> run, final Launcher launcher, fin
         VSphereLogger.vsLogger(jLogger, "Preparing reconfigure: CPU");
         spec.setNumCPUs(Integer.valueOf(expandedCPUCores));
         spec.setNumCoresPerSocket(Integer.valueOf(expandedCoresPerSocket));
+        spec.setCpuAllocation(resAllInfo);
```

Comment:
If the user hasn't set `cpuLimitMHz` then `spec.setCpuAllocation(...)` should not be called.
i.e. preserve existing default behavior.